wiki/osu!wiki_contribution_guide/en.md

```diff
@@ -0,0 +1,724 @@
+# How to Contribute to the osu!wiki
+
+Thanks for your interest in making osu!wiki better!
+As you can see, everything has been moved to [GitHub](https://github.com/ppy/osu-wiki).
+
+So here's a few things you should have in mind to get you started for contributing to the osu!wiki.
+
+## Disclaimer
+
+This quick-start guide was intended for newcomers not familiar in the workflow of GitHub.
+If you have previous experience in using GitHub, you are free to skip this guide.
+Do note that there are other ways/shortcuts in solving the problem; this guide is simply one of the many ways of solving the problem.
+
+When starting out, it is best to start on GitHub's way first, and slowly learn the terminal Git way properly in the meantime.
+Knowing how GitHub works will make the transition to terminal Git slightly smoother when you know what you are doing.
+
+For complete reference and help with using GitHub, please visit [GitHub Help](https://help.github.com/ "GitHub Help").
+
+For a styling guide when writing a page, check out [Article Style Guide](/wiki/Article_Style_Guide "Article Style Guide") instead.
+
+## Prerequisites
+
+To be able to contribute to osu!wiki, you must have the following:
+
+- A [GitHub account](https://help.github.com/articles/signing-up-for-a-new-github-account/).
+- Knowledge in [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet), specifically [GitHub-Flavoured Markdown(GFM)](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/ "GitHub Help on Writing on GitHub").
+- [Understanding the GitHub's workflow](https://help.github.com/articles/what-is-a-good-git-workflow/ "GitHub Help on GitHub workflow").
+  - A simplified visual explanation can be found [here](https://guides.github.com/introduction/flow/).
+- Able to write in English, preferably British English for consistency.
+  - Everything not related to the page content itself (commit messages, pull requests, filenames, etc) **MUST** be written in English.
+
+Optionally, if you plan to make changes locally:
+
+- A text editor with Markdown syntax highlighting.
+  - We either use Microsoft's [Visual Studio Code](https://code.visualstudio.com "Visual Studio Code") or GitHub's [Atom](https://atom.io "Atom").
+  - Any text editor will suffice if you do not need the syntax highlighting.
```

Comment:
remove this, it seems redundant to say, _must include this_... then say _oh, btw, it really doesn't :p_